### PR TITLE
checks: Fix E402 Flake8 in for wxGUI Sphinx doc

### PR DESCRIPTION
**Pull request by arohanajit**
Fixed `E402` error in `gui/wxpython/docs/wxgui_sphinx` - moved import module to the top

### CODE REVIEW

**Comment by echoix** [comment]
What I see here is that the check for gisbase was made probably to prevent loading grass.script core if not set. However, if it were that important, it would've been a pattern we see everywhere.

Does @wenzeslaus know if we are more robust now and the import can be done and the check after? If so, that PR is fine to merge!

**Review by wenzeslaus — CHANGES_REQUESTED** [review]
It seems to be that this code was never doing anything useful. Adding etc/python/grass to sys.path does not make grass.script importable which is clearly the intention here (an import after a sys.path.append call).

You can just remove the whole block with GISBASE. Who wants to compile the documentation, needs to have _grass_ on path - that's not such an unexpected requirement at this point. GISBASE may not even be needed on the other hand.

If I'm not mistaken, we don't compile wxGUI doc anywhere on servers, so this should be very safe to mess with.

**Comment by arohanajit** [comment]
Got it, I'll remove that code block. I'll also remove 'sys' and 'os' library imports since that was the only place where they are used and will highlight new `flake8` errors

File: gui/wxpython/docs/wxgui_sphinx/conf.py [hunk 1]
```diff
@@ -10,22 +10,14 @@
...
-import sys
-import os
 from datetime import date
 import string
 from shutil import copy
+from grass.script import core
 
 # If extensions (or modules to document with autodoc) are in another directory,
 # add these directories to sys.path here. If the directory is relative to the
 # documentation root, use os.path.abspath to make it absolute, like shown here.
```
Comment (wenzeslaus) [inline gui/wxpython/docs/wxgui_sphinx/conf.py:20]: The comment should go with that. It does not fit much with the code and what is relevant is likely in Sphinx doc.

**Review by wenzeslaus — CHANGES_REQUESTED** [review]

**Review by wenzeslaus — APPROVED** [review]

**Comment by echoix** [comment]
I fixed the merge conflict

**Review by echoix — APPROVED** [review]

**Pull request merged**